### PR TITLE
Clean up BuildFiles under Calibration/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
hold

**Comment by fwyzard** [comment]
unhold

**Comment by fwyzard** [comment]
@cmsbuild, please test

File: Calibration/EcalAlCaRecoProducers/plugins/BuildFile.xml [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-# for FED list selector
-<use   name="DataFormats/EgammaCandidates"/>
-<use   name="CondFormats/SiPixelObjects"/>
-<use   name="CalibFormats/SiStripObjects"/>
-<use   name="CalibFormats/HcalObjects"/>
-<use   name="Geometry/EcalMapping"/>
-<use   name="Geometry/CaloGeometry"/>
-<use   name="CalibTracker/Records"/>
-<flags   EDM_PLUGIN="1"/>
+<use name="clhep"/>
```
Comment (silviodonato) [inline Calibration/EcalAlCaRecoProducers/plugins/BuildFile.xml:1]: @fwyzard

Comment (fwyzard) [reply Calibration/EcalAlCaRecoProducers/plugins/BuildFile.xml:1]: removed the comment and sorted

**Pull request merged**